pkg/http/api/v1/handler.go

```diff
@@ -21,7 +22,7 @@ const (
 	JSONRequestMimeType           = "application/json"
 	HarborVulnReportv1MimeType    = "application/vnd.scanner.adapter.vuln.report.harbor+json; version=1.0"
 	RawVulnReportMimeType         = "application/vnd.scanner.adapter.vuln.report.raw+json"
-	DockerImageMimeType           = "application/vnd.oci.image.manifest.v1+json"
+	DockerImageMimeType           = "application/vnd.oci.image.manifest.v2+json"
```

Comment:
Same here, I just want positive confirmation that it should replace the v1 rather than add support for v2.

Reply:
This is a straight replacement. It offers backward compatibility per my discussions with Steven